src/app/modules/rup/components/ejecucion/prestacionEjecucion.component.ts

```diff
@@ -571,18 +573,18 @@ export class PrestacionEjecucionComponent implements OnInit, OnDestroy {
 
             if (registro.relacionadoCon && registro.relacionadoCon.length > 0) {
                 registro.relacionadoCon.forEach((registroRel, key) => {
-                    let esRegistro = this.prestacion.ejecucion.registros.find(r => {
+                    let registroAux = this.prestacion.ejecucion.registros.find(r => {
                         if (r.id) {
-                            return r.id === registroRel;
+                            return r.id === registroRel.id;
                         } else {
-                            return r.concepto.conceptId === registroRel;
+                            return r.concepto.conceptId === registroRel.concepto.conceptId;
                         }
                     });
                     // Es registro RUP o es un concepto puro?
-                    if (esRegistro) {
-                        registro.relacionadoCon[key] = esRegistro;
+                    if (registroAux) {
+                        registro.relacionadoCon[key] = { id: registroAux.id };
```

Comment:
Array de ID, no array de objetos con un campo ID
```suggestion
                        registro.relacionadoCon[key] =  registroAux.id;
```